app/config/wallabag.yml

```diff
@@ -163,6 +163,20 @@ wallabag_core:
         - 'OpenDyslexicRegular'
         - 'Oswald'
 
+    client_side_rendered_sites:
+        proxy_url: https://single-file-proxy.your-domain.tld
+        proxy_all: true
+        sites_to_proxy:
+            # - https://www.blast-info.fr
+            # - https://www.arretssurimage.net
```

Comment:
Same comment about `app/config/parameters.yml.dist`

Reply:
Would it be better as an application-level config, something like the Site credential management?

Reply:
Of course

Reply:
This point remains unfixed. I'll have a look at this app-level integration now.